tests/helpers.py

```diff
@@ -53,6 +53,8 @@ def ip_to_device_id_map(device_group, devauth=devauth):
         # Match them.
         identity_to_id = {}
         for dev in devauth_devices:
+            # Strip the array from the reported identity
+            dev["identity_data"]["mac"] = dev["identity_data"]['mac'][0]
```

Comment:
I am concerned about this (from the client point of view, not the test). Was it intentional? Then we should have a changelog entry. Was it not intentional? Then we should revert.

FYI @kacf

Reply:
Yes, we should probably discuss this.

I guess it's not wrong in the sense that an array of one member is the same as one element, I guess we could ask @kjaskiewiczz for his opinion here.

Crux of the matter KJ is that we now send the identity data as an array of one member (by default).

Reply:
Yes, the backend will be happy with it but some custom scripts (like this exact test) will require adjustments by the user.

Reply:
What custom scripts? This is simply a difference in how we marshal and send it to the server. The identity script has not changed at all

Reply:
If I understand correctly, we are simply using inventory data (where array is fine) to reconstruct identity data (where array is not fine). This will never be done by any user, so should be fine, right?

Reply:
I just don't get why this changed with the new client, sorry.

But you both seem pretty convinced that it is okey, so I won't insist further.